cmd/statCalc/statCalc.go

```diff
@@ -6,202 +6,148 @@ import (
-			" -poisson=lambda. Defines a poisson distribution with rate parameter lambda. Ex Usage: -poisson=4 4\n" +
-			" -beta=alpha,beta. Defines a beta dsitribution with paramters alpha and beta. Ex Usage: -beta=5,5 0.2\n" +
-			" -gamma=alpha,beta. Defines a gamma distribution with parameters alpha and beta. Ex Usage: -gamma=4,4 6\n" +
-			" -sampleAfs=alpha,numSamples,maxSampleDepth,bins,xLeft,xRight. Provides a list of values sampled from an allele frequency spectrum with selection parameter alpha.\n" +
-			"sampleAFS will return numSamples many values between xLeft and xRight. Bins and maxSampleDepth are performance and accuracy options, suggested values are 1000 and 1000, respectively.\n" +
-			"After defining a distribution, one float64 argument returns the function density at that value. Ex usage: -sampleAfs=0.02,200,1000,1000,0.001,0.999,false\n" +
-			"sampleBeta=alpha,beta,numSamples. Provides a list of values sampled from the beta distribution with a selected alpha and beta parameter.\n" +
-			"sampleGamma=alpha,beta,numSample. Provides a list of values sampled from the gamma distribution with a selected alpha and beta parameter.\n" +
-			"sampleNormal=mu,sigma,numSamples. Provides a list of values sampled from the normal distribution with a selected mu and sigma parameter.\n" +
-			"For discrete distributions, two arguments will evaluate the sum between two input values.\n" +
-			"For the binomial distribution summation, the second argument can be set to n or N to evaluate the entire right tailed sum.\n" +
-			"For continuous distributions, two arguments will evaluate an integral between the two input values with the defined distribution as the integrand.\n")
-}
-
-func main() {
-	var Normal *string = flag.String("normal", "", "")
-	var Binomial *string = flag.String("binomial", "", "")
-	var Poisson *string = flag.String("poisson", "", "")
-	var Beta *string = flag.String("beta", "", "")
-	var Gamma *string = flag.String("gamma", "", "")
-	var SampleAfs *string = flag.String("sampleAfs", "", "")
-	var SampleBeta *string = flag.String("sampleBeta", "", "")
-	var SampleGamma *string = flag.String("sampleGamma", "", "")
-	var SampleNormal *string = flag.String("sampleNormal", "", "")
-	var randSeed *bool = flag.Bool("randSeed", false, "Uses a random seed for the RNG.")
-	var setSeed *int64 = flag.Int64("setSeed", -1, "Use a specific seed for the RNG.")
-
-	flag.Usage = usage
-	log.SetFlags(log.Ldate | log.Ltime | log.Lshortfile)
-	flag.Parse()
-
-	MultipleOptionErrorCheck(Normal, Binomial, Poisson, Beta, Gamma, SampleAfs, SampleBeta, SampleGamma, SampleNormal)
-	common.RngSeed(*randSeed, *setSeed)
-	if *Normal != "" {
-		words := strings.Split(*Normal, ",")
+func statCalc(s Settings) {
+	MultipleOptionErrorCheck(s.Normal, s.Binomial, s.Poisson, s.Beta, s.Gamma, s.SampleAfs, s.SampleBeta, s.SampleGamma, s.SampleNormal)
+	common.RngSeed(s.RandSeed, s.SetSeed)
```

Comment:
Did we come to a conclusion on the idea of removing random seeding in most places?

Reply:
yes but that's a major change. Should be its own request